packages/subcommands/src/lib/SubCommandEntryMethod.ts

```diff
@@ -0,0 +1,9 @@
+import { SubCommandEntry } from './SubCommandEntry';
+
+export class SubCommandEntryMethod extends SubCommandEntry {
+	public run(context: SubCommandEntry.RunContext): unknown {
+		const method = Reflect.get(context.command, this.output);
+		if (method) return Reflect.apply(method, context.command, [context.message, context.args, context.context]);
+		throw new ReferenceError(`The method '${this.input}' does not exist for the command '${context.command.name}'.`);
```

Comment:
Should probably throw a proper `err` here. Same for all other errors. Looks like Kyra was lazyass mode in Skyra and didn't bother with i18n errors.